packages/agent-app-sdk/src/widgets/shared/page-data.ts

```diff
@@ -0,0 +1,33 @@
+import { WidgetMixin } from '@livechat/widget-core-sdk';
+
+export interface IPageData {
+  queryParams: object;
+}
+
+export interface IPageDataApi {
+  getPageData(): IPageData | null;
+}
+
+export interface IPageDataEvents {
+  page_data: IPageData;
+}
+
+export const withPageData: WidgetMixin<
+  IPageDataApi,
+  IPageDataEvents
+> = widget => {
+  let pageData = null;
+
+  function onPageData(data: IPageData) {
+    pageData = data;
+  }
+
+  widget.on('page_data', onPageData);
```

Comment:
```suggestion
  widget.on('page_data', (data: IPageData) => {
    pageData = data;
  });
```

if function onPageData is not used anywhere else it can be done like that, It got me bit confused as below there is function returned with the same name

Reply:
This solution was based on [customer-profile.ts](https://github.com/livechat/products-sdk/blob/0d7758884e4ecde7445de23ebaa2108c8f6b3c97/packages/agent-app-sdk/src/widgets/shared/customer-profile.ts#L32). 
Those two functions are almost the same, so I wanted to make this code consistent with solution that already exists :)

Reply:
alright, then Im resolving this